provider/machine_pool_resource.go

```diff
@@ -142,6 +151,29 @@ func (t *MachinePoolResourceType) GetSchema(ctx context.Context) (result tfsdk.S
 				},
 				Optional: true,
 			},
+			"availability_zones": {
+				Description: "The availability zones in which to place the machine pool",
```

Comment:
The description in rosa cli is 
"Select availability zone to create a single AZ machine pool for a multi-AZ cluster"
I think we would like to be align to rosa cli with the purpose of that attribute